integration/test/basic/basic_test.go

```diff
@@ -4,58 +4,53 @@ package basic
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"testing"
+	"time"
 
-	"github.com/giantswarm/appcatalog"
-	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/backoff"
 	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/micrologger"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestHelm(t *testing.T) {
+// TestSecrets ensures that kiam-server and kiam-agent created
+func TestSecrets(t *testing.T) {
 	ctx := context.Background()
+	var err error
 
-	// Install cert-manager so kiam certs can be issued.
-	err := installCertManager(ctx, helmClient, l)
+	err = checkReadyDaemonset(ctx)
 	if err != nil {
-		t.Fatalf("%#v", err)
-	}
-
-	err = ba.Test(context.Background())
-	if err != nil {
-		t.Fatalf("%#v", err)
+		t.Fatalf("could not get kiam-agent & kiam-server: %v", err)
 	}
 }
 
-func installCertManager(ctx context.Context, helmClient helmclient.Interface, logger micrologger.Logger) error {
-	tarballURL, err := appcatalog.GetLatestChart(ctx, catalogURL, certManagerAppName, "")
-	if err != nil {
-		return microerror.Mask(err)
-	}
+func checkReadyDaemonset(ctx context.Context) error {
```

Comment:
```suggestion
func checkDaemonsetExists(ctx context.Context) error {
```

Testing cloud provider apps in KIND is tricky. Good to change this but can you also update the func name and logging?

Reply:
Sure I'm updating those now.